### ISSUE TITLE
Yarn lock and package.json

### ISSUE THREAD

**Issue by isaacabraham** [slogsdon/parcel-plugin-fable#12]
Are the two files in sync? For example, there are no dependencies, just two dev dependencies for parcel, yet the lock file contains bable and fable etc. etc.?

**Comment by slogsdon** [comment]
Yes, they should be in sync. The latest version (1.5.0) does have two dependencies (babylon-walk and fable-utils), viewable in both [package.json](https://github.com/slogsdon/parcel-plugin-fable/blob/master/package.json#L16-L17) and on [npmjs.com](https://www.npmjs.com/package/parcel-plugin-fable).
